simcon_project/simcon_project/settings.py

```diff
@@ -143,8 +144,20 @@
 LOGOUT_REDIRECT_URL = 'Login'
 LOGIN_URL = 'Login'
 
-# Bootstrap Template for Django Tables 
+# Bootstrap Template for Django Tables
 DJANGO_TABLES2_TEMPLATE = "django_tables2/semantic.html"
 MEDIAFILES_DIRS = [
     os.path.join(BASE_DIR, 'static/media')
 ]
+MEDIA_URL = '/media/'
+
+
+#we will need to hide this in the final version
+SENDGRID_API_KEY = os.getenv('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')
+EMAIL_HOST = 'smtp.sendgrid.com'
+EMAIL_PORT = 587
+EMAIL_HOST_USER = 'apikey'
+EMAIL_HOST_PASSWORD = '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'
```

Comment:
I know its base 64 encoded but feels unsfe

Reply:
yeah. I wasn't sure if hiding this should go on this ticket or the one where we talked about adding multiple email accounts for us to send as many emails as we want. should I try and do it for this ticket or wait for it to be on the other one?